src/herder/test/HerderTests.cpp

```diff
@@ -1066,8 +1072,12 @@ TEST_CASE("surge pricing", "[herder][txset]")
 
 static void
 testSCPDriver(uint32 protocolVersion, uint32_t maxTxSize, size_t expectedOps,
-              bool checkHighFee, bool withSCPsignature)
+              bool checkHighFee, bool withSCPsignature,
+              bool const expectNewClosetimeBehavior)
 {
+    using CloseTime = decltype(StellarValue::closeTime);
```

Comment:
This doesn't add clarity, because you are using names different from our convention. When I read `CloseTime`, my mind questions what type that is rather than automatically jumping to `TimePoint`. Is there a good reason not to use `TimePoint` and `UpgradeType`, like we do everywhere else?

Reply:
`StellarValue::upgrades` isn't of type `UpgradeType`; it's of type `xdr::xvector<UpgradeType,6>`, which made me a little queasy to write in a bunch of places (hence the `using`) -- or even one, for that matter (hence the `decltype`).  After that I just felt strange treating the `closeTime` and `upgrades` fields differently as far as whether I was inferring their types or not.

I've gotten rid of the `CloseTime` in favor of `TimePoint`.  Do you definitely also want me to replace `SVUpgrades` with `xdr::xvector<UpgradeType,6>`?